routes/v3/covid-19/apiVariants.js

```diff
@@ -0,0 +1,41 @@
+// *from here* This model comes from apiGov.js:
+
+// eslint-disable-next-line new-cap
+const router = require('express').Router();
+
+const nameUtils = require('../../../utils/nameUtils');
+const { wordToBoolean } = require('../../../utils/stringUtils');
+const { redis, keys } = require('../../instances');
+
+router.get('/v3/covid-19/variants/countries/:country?', async (req, res) => {
+	const { allowNull } = req.query;
+	const { country: countryName, yearWeek, variant  } = req.params;
```

Comment:
I don't see the yearWeek and variant parameters being used

Reply:
I put them there since I wanted the API had to have these parameters, but I think it's better to remove them and later when they're created added again.